src/Halogen/Internal/VirtualDOM.purs

```diff
@@ -128,7 +128,19 @@ foreign import vnode
   \  return function(props) {\
   \    return function(children) {\
   \      var VirtualNode = require('virtual-dom/vnode/vnode');\
-  \      return new VirtualNode(name, props, children);\
+  \      var props1 = {\
+  \        attributes: {}\
+  \      };\
+  \      for (var key in props) {\
```

Comment:
Yep. I thought we were kind of heading that way for all PS code anyway. I know there's ES5 stuff that crept into `Array` already at least.